sdk/azurestackhci/Azure.ResourceManager.Hci/README.md

```diff
@@ -43,6 +43,299 @@ Documentation is available to help you learn how to use this package:
 - [Authentication](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/README.md).
 
 ## Examples
+#### Prerequisites
```

Comment:
Please follow https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/README.md?plain=1#L44 to add samples by using code Snippet which can make sure the code is compilable.

Reply:
Resolved, please review

Reply:
You need add a **cs file** with the sample code, like this https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/samples/DocSamples/Readme.cs, then add the `C# Snippet` link with the corresponding code snippet, then run `eng\scripts\Update-Snippets.ps1 azurestackhci` to update the code from the **cs file**.